index.js

```diff
@@ -0,0 +1,16 @@
+let names=[]
+function myFunction()
+{
+ let x=document.getElementById("box");
+  names.push(document.getElementById("name").value);
+ x.innerHTML=names.join('<br/><br/>'); 
```

Comment:
Your logic doesn't **add the names as a list of names**.

The expectation is that each name that is added should be an item in a list so that it is rendered as a list, and is accessible as shown below:

```
<ol>
  <li>Name 1</li>
  <li>Name 2</li>
<ol>

```

Reply:
Alright, Sir.
Thank you for your review and feedback on my PR. Your attention to detail and keen eye for errors really helped me catch mistakes that I missed, and I'm grateful for your input. I'll do well to refactor and submit a PR